core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -286,11 +286,14 @@ <h3>Roles</h3>
                   </span>
                 </div>
 
-                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save">
+                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save" ng-click="checkForTitle($event)">
                 <button type="button" class="btn btn-default" ng-click="closeEditRolesForm()"
                         ng-show="isRolesFormOpen">
                   Cancel
                 </button>
+                <p ng-show="hasTitleBeenAdded">
+                        *Please add title and try again
```

Comment:
Can we make the message red? Just to clarify that it is an error. Currently this looks more like some additional info (like the huge paragraph above) , that might be missed by the user.

Reply:
I thought of making it red, but I felt its better I keep normal black, and check with someone if I should make it red or black, 
Ill disable the Button (save) instead of allowing user to click and prevent submit.
Doing it rn, and for the travis Errors, Ill check why that is happening.

Reply:
@nithusha21 , is it okay if I use a little bit of dom-manipulation (add a keydown event on the Title input) Im not sure how to do it without that. I did try a few things I found after reading through angular docs, but most did not give the result needed here.

Reply:
You can add an angular element, and use the ng-hide or ng-show attribute, which is controlled by a boolean expression.